src/handlers/comment/handlers/table.ts

```diff
@@ -4,21 +4,27 @@ export const tableComment = ({
   multiplier,
   reason,
   bounty,
+  isBountyStale,
+  days,
 }: {
   deadline: string;
   wallet: string;
   multiplier?: string;
   reason?: string;
   bounty?: string;
+  isBountyStale?: boolean | null;
```

Comment:
Why do we need 3 states for the `isBountyStale` variable (`true | false | null`)? Can't simply use the `boolean` type?

Reply:
Ok, so there are 3 states:
1. `stale-bounty-time = 0` => nothing is [rendered](https://github.com/ubiquity/ubiquibot/blob/7fd7d3b83aaab0b19673d9c601937d9fdeb3f18c/src/handlers/comment/handlers/table.ts#L23)
2. `stale-bounty-time = 30d` **and** threshold reached => "warning" row is [rendered](https://github.com/ubiquity/ubiquibot/blob/7fd7d3b83aaab0b19673d9c601937d9fdeb3f18c/src/handlers/comment/handlers/table.ts#L25)
3. `stale-bounty-time = 30d` **and** threshold is **not** reached => nothing is [rendered](https://github.com/ubiquity/ubiquibot/blob/7fd7d3b83aaab0b19673d9c601937d9fdeb3f18c/src/handlers/comment/handlers/table.ts#L26)